src/components/PostPreview/PostPreview.tsx

```diff
@@ -33,57 +33,25 @@ export const PostPreview: React.FC<PostPreviewProps> = ({
   hasMask,
   isRelated,
   isLoading = false,
-  ...boxProps
-}) => {
-  const titleBlock = hasMask ? (
-    (title as React.JSX.Element)
-  ) : (
-    <Link {...link} data-internal-link={isRelated ? 'relatedPost' : 'post'}>
-      {title}
-    </Link>
-  );
-  return (
-    <Box
-      as="article"
-      className={classNames('post-preview', { 'post-preview--mask': hasMask }, { 'post-preview--related': isRelated })}
-      {...boxProps}
-    >
-      <Skeleton isLoading={isLoading}>
-        <Heading as="h2" color="amaranth" size="s">
-          {contentType === ContentTypeEnum.TUTORIAL ? (
-            <Flex gap="xxs">
-              <TutoTag />
-              {titleBlock}
-            </Flex>
-          ) : (
-            titleBlock
-          )}
-        </Heading>
-      </Skeleton>
-      <Skeleton isLoading={isLoading} style={{ height: 75 }}>
-        <Text size="s" className="post-preview__excerpt">
-          {excerpt}
-        </Text>
-      </Skeleton>
-      <Box mt={{ xs: 'xs', md: 's' }} textSize="xs">
-        <Skeleton isLoading={isLoading} display="inline-block" style={{ width: 100 }}>
-          {date && <Text as="span">{date}</Text>}
-        </Skeleton>
-        <SeparatorCircle />
-        <Skeleton isLoading={isLoading} display="inline-block" style={{ width: 50 }}>
-          {readingTime && <Text as="span">{`${readingTime}mn`}</Text>}
-        </Skeleton>
-        <SeparatorCircle />
-        <Skeleton isLoading={isLoading} display="inline-block" style={{ width: 100 }}>
-          {authors &&
-            authors.map((author, authorIndex) => (
-              <Text key={author.username} as="span">
-                {author.name}
-                {authorIndex !== authors.length - 1 ? ' & ' : ''}
-              </Text>
-            ))}
-        </Skeleton>
-      </Box>
-    </Box>
-  );
-};
+  isHighlighted = true,
```

Comment:
```suggestion
  isHighlighted = false,
```
La valeur est à false par défaut